backend/hitas/models/address.py

```diff
@@ -0,0 +1,20 @@
+from django.contrib import admin
+from django.db import models
+
+from .codes import PostalCode
+
+
+class Address(models.Model):
+    street_address = models.CharField(max_length=1024)
+    postal_code = models.ForeignKey(PostalCode, on_delete=models.PROTECT)
+
+    class Meta:
+        verbose_name_plural = 'Addresses'
```

Comment:
Even if Django Admin should only support english (probably?) it's still smart to use `gettext_lazy` on verbose names
(Also add the singular form here while you're at it? 😉)